.github/workflows/function-composition.yml

```diff
@@ -60,23 +63,18 @@ jobs:
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
 
       - name: Set up Docker Buildx
-        id: buildx
         uses: docker/setup-buildx-action@v1
 
       - name: Build and push
-        id: docker_build
         uses: docker/build-push-action@v2
         with:
-            context: ./configs/golang/
-            file: ./configs/golang/Dockerfile
+            context: ./configs/docker-images/${{ matrix.image }}
             push: true
-            tags: |
-              ${{ secrets.DOCKER_HUB_USERNAME }}/golang:latest
-              ${{ secrets.DOCKER_HUB_USERNAME }}/${{ env.VHIVE_GOLANG_VERSION }}
+            tags: ${{ secrets.DOCKER_HUB_USERNAME }}/${{ matrix.image }}:latest
```

Comment:
please push a tag specifying golang version here along with `:latest` like before to simplify rollbacks if necessary.